specification/purview/data-plane/Azure.Analytics.Purview.Scanning/stable/2023-09-01/scanningService.json

```diff
@@ -0,0 +1,10292 @@
+            "$ref": "./examples/Filters_CreateOrReplace.json"
+          }
+        }
+      }
+    },
+    "/integrationruntimes": {
+      "get": {
+        "tags": [
+          "IntegrationRuntimes"
+        ],
+        "description": "Lists integration runtimes in an account.",
+        "operationId": "IntegrationRuntimes_ListByAccount",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/api-version"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success.",
+            "schema": {
+              "$ref": "#/definitions/IntegrationRuntimeList"
+            }
+          },
+          "default": {
+            "description": "An error response received from the Scanning Service.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponseModel"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "IntegrationRuntimes_ListByAccount": {
+            "$ref": "./examples/IntegrationRuntimes_ListByAccount.json"
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/integrationruntimes/{integrationRuntimeName}": {
+      "get": {
+        "tags": [
+          "IntegrationRuntimes"
+        ],
+        "description": "Gets an integration runtime.",
+        "operationId": "IntegrationRuntimes_Get",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/integrationRuntimeName"
+          },
+          {
+            "$ref": "#/parameters/api-version"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success.",
+            "schema": {
+              "$ref": "#/definitions/IntegrationRuntime"
+            }
+          },
+          "default": {
+            "description": "An error response received from the Scanning Service.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponseModel"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "IntegrationRuntimes_Get": {
+            "$ref": "./examples/IntegrationRuntimes_Get.json"
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "IntegrationRuntimes"
+        ],
+        "description": "Deletes an integration runtime.",
+        "operationId": "IntegrationRuntimes_Delete",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/integrationRuntimeName"
+          },
+          {
+            "$ref": "#/parameters/api-version"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success."
+          },
+          "204": {
+            "description": "Success."
+          },
+          "default": {
```

Comment:
nit, consider adding `x-ms-error-code` to error response https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md#rest-error-code-header